### PR TITLE
Updated codeowners & added config warning comment

### PR DESCRIPTION
**Pull request by KuphJr**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -46,8 +46,16 @@
 
 /core/services/webhook @samsondav
 
-# Chainlink Functions related services
+# Chainlink Functions
+core/services/functions @bolekk @justinkaseman @KuphJr @pinebit
```
Comment (bolekk) [inline CODEOWNERS:50]: These all should have a team name, like for VRF directories.

**Review by bolekk — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (KuphJr) [reply CODEOWNERS:50]: I can't find a team tag for us (I see vrf-team, but not for Functions).  I recall Brad saying the was going to set up a team tag for us, but IDK if that actually happened.

Comment (justinkaseman) [reply CODEOWNERS:50]: We don't have a team tag @bolekk

**Review by justinkaseman — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (justinkaseman) [reply CODEOWNERS:50]: We can amend this later once we do

Comment (bolekk) [reply CODEOWNERS:50]: Let's get one then!

**Review by bolekk — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (KuphJr) [reply CODEOWNERS:50]: We can get a proper team before I merge this PR.  No urgency to get this in.

**Review by pinebit — DISMISSED** [review]

**Review by pinebit — APPROVED** [review]

**Pull request merged**